### PR TITLE
Task popup

### PR DESCRIPTION
**Pull request by ajhswan**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+    <link rel='stylesheet' type='text/css' href='normalize.css'/>
+    <link rel='stylesheet' type='text/css' href='styles.css' />
+</head>
+
+<body>
+    <h1> Receipts </h1>
+    <div class='receiptsContainer'>
+        <?php echo $receipts;?>
+    </div>
+    <div class='box'>
```
Comment (markandmarkup) [inline index.php]: I'd give classes more descriptive names if possible.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        <a class='button' href='#popup'>Add Receipt</a>
+    </div>
+
+    <div id='popup' class='overlay'>
+        <div class='popup'>
+            <h2> Add New Receipt </h2>
+            <a class='close' href='#'>&times;</a>
+            <div class='content'>
+                <form action='index.php' method='post'>
+                    <div class='form'>
```
Comment (markandmarkup) [inline index.php:51]: Again, more descriptive class name. '.form' would be confusing reading the css when it refers to a div

File: index.php [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+                        <h4>Supplier:</h4>
```
Comment (markandmarkup) [inline index.php]: Instead of h4, these should be labels. Add 'for' attribute, and give the input a corresponding id.

File: styles.css [hunk 4]
```diff
@@ -0,0 +1,177 @@
...
+}
+
+.popup .content {
+    max-height: 60%;
+    overflow: auto;
+
+}
+/*.popup h4 {*/
+/*    display: inline;*/
+/*}*/
```
Comment (markandmarkup) [inline styles.css]: Remove unused

File: styles.css [hunk 5]
```diff
@@ -0,0 +1,177 @@
+body {
+    background-color: gold;
+}
+
+.receipt {
+    float: left;
+    min-height: 250px;
```
Comment (markandmarkup) [inline styles.css:7]: Check this does something

File: styles.css [hunk 6]
```diff
@@ -0,0 +1,177 @@
...
+    text-decoration: none;
+    color: #333333;
+}
+
+.popup .close:hover {
+    color: cornflowerblue;
+}
+
+.popup .content {
+    max-height: 60%;
```
Comment (markandmarkup) [inline styles.css]: Check this does something

File: styles.css [hunk 7]
```diff
@@ -0,0 +1,177 @@
...
+    overflow: auto;
+
+}
+/*.popup h4 {*/
+/*    display: inline;*/
+/*}*/
+.popup .form {
+    display: inline-block;
+    margin: 0;
```
Comment (markandmarkup) [inline styles.css]: Check these 2 do something

**Review by markandmarkup — COMMENTED** [review]
Nice work, a few minor changes I'd make, but not a big job

File: index.php (hunk 1 above)
Comment (ajhswan) [reply index.php]: changed to buttonBox

File: index.php (hunk 3 above)
Comment (ajhswan) [reply index.php]: Done

File: styles.css (hunk 4 above)
Comment (ajhswan) [reply styles.css]: done

File: styles.css (hunk 6 above)
Comment (ajhswan) [reply styles.css]: removed

File: styles.css (hunk 7 above)
Comment (ajhswan) [reply styles.css]: removed

**Pull request merged**